core/server/master/src/test/java/alluxio/master/metrics/MetricsMasterTest.java

```diff
@@ -102,12 +102,12 @@ public void testMultiValueAggregator() throws Exception {
     mMetricsMaster.addAggregator(
         new SingleTagValueAggregator("metric", MetricsSystem.InstanceType.WORKER, "metric", "tag"));
     List<Metric> metrics1 = Lists.newArrayList(
-        Metric.from("worker.metric.192_1_1_1.tag:v1", 10, MetricType.GAUGE),
-        Metric.from("worker.metric.192_1_1_1.tag:v2", 20, MetricType.GAUGE));
+        Metric.from("worker.metric.tag:v1.192_1_1_1", 10, MetricType.GAUGE),
+        Metric.from("worker.metric.tag:v2.192_1_1_1", 20, MetricType.GAUGE));
     mMetricsMaster.workerHeartbeat("192_1_1_1", metrics1);
     List<Metric> metrics2 = Lists.newArrayList(
-        Metric.from("worker.metric.192_1_1_2.tag:v1", 1, MetricType.GAUGE),
-        Metric.from("worker.metric.192_1_1_2.tag:v2", 2, MetricType.GAUGE));
+        Metric.from("worker.metric.tag:v1.192_1_1_2", 1, MetricType.GAUGE),
+        Metric.from("worker.metric.tag:v2.192_1_1_2", 2, MetricType.GAUGE));
```

Comment:
Do we want to be testing `MetricType.COUNTER`?

Reply:
Will add the tests after finishing and testing out all those PRs. For the MetricsStore, based on my findings today, I still need to modify it.

One of my TODO is don't send and aggregated throughput meter one minute rate in master. This is because the client needs five seconds to have non-zero one minute rate and most clients don't live that long. Plan to change to calculated throughput directly on master side.